src/util/settings/connection_client.cc

```diff
@@ -30,75 +30,116 @@ QString localhost("127.0.0.1");
 QString default_database_file("veles.vdb");
 int default_server_port = 3135;
 bool default_shut_down_server = true;
+QString default_profile("local server");
+
+QString currentProfile() {
+  QSettings settings;
+  return settings.value("profile", default_profile).toString();;
+}
+
+void setCurrentProfile(QString profile) {
+  QSettings settings;
+  settings.setValue("profile", profile);
+}
+
+void removeProfile(QString profile) {
```

Comment:
Mhm, I thought that invalid QVariant (default constructed) can't be converted to map, but it seems that I was wrong.